makelib/golang.mk

```diff
@@ -226,12 +235,12 @@ go.generate:
 # Common Targets
 
 build.init: go.init
-build.check: go.fmt
-build.check.platform: go.vet
+build.check: go.lint
 build.code.platform: go.build
 clean: go.clean
 distclean: go.distclean
-lint: go.lint
+lint.init: go.init
+lint.run: go.lint
```

Comment:
I think you would need to modify `common.mk` to hook the new `lint.*` targets in.

Reply:
That does not seem to be the case. I've tested this and it seems to work as expected when I run  `make build`, `make build.all` or `make lint` using this branch as a submodule of Crossplane.

Reply:
To be clear - these aren't new targets - `lint.init` and `lint.run` were both already declared in `common.mk`.

Reply:
Aaah I see them now.